docs/resources/federation_federated_block.md

```diff
@@ -0,0 +1,67 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_federation_federated_block Resource - terraform-provider-bloxone"
+subcategory: ""
+description: |-
+  
+---
+
+# bloxone_federation_federated_block (Resource)
+
+
+
+## Example Usage
+
+```terraform
+resource "bloxone_federation_federated_realm" "example" {
+  name = "example_federation_federated_realm"
+}
+
+resource "bloxone_federation_federated_block" "example" {
+  name                       = "example_federation_federated_block"
+  federation_federated_realm = bloxone_federation_federated_realm.example.id
+  cidr                       = 24
+  address                    = "10.10.0.0"
+
+  //tags
+  tags = {
+    key1 = "value1"
+    key2 = "value2"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `cidr` (Number) The CIDR of the federated block. This is required, if _address_ does not specify it in its input.
+- `federation_federated_realm` (String) The resource identifier.
+
+### Optional
+
+- `address` (String) The address of the subnet in the form “a.b.c.d”
+- `allocation_v4` (Attributes) The percentage of the Federated Block’s total address space that is consumed by Leaf Terminals. (see [below for nested schema](#nestedatt--allocation_v4))
```

Comment:
everything under `allocation_v4` is read only, you might as well make this read only as well.

Reply:
done